docs/guides/bs5migration/index.md

```diff
@@ -382,3 +382,52 @@ shift-color($primary, -20%);
 The `theme-color-level()` has been changed to `color-level()` and then subsequently removed and replaced by scale-color().
 In the stable 5.0 the final decision was to adopt `shift-color()` so we will use this function in the bridge file.
 :::
+
+### Screen reader utilities
+
+"Screen reader" classes are now "visually hidden" classes.
+
+- Renamed .sr-only and .sr-only-focusable to .visually-hidden and .visually-hidden-focusable.
+- Renamed sr-only() and sr-only-focusable() mixins to visually-hidden() and visually-hidden-focusable().
```

Comment:
- Renamed `.sr-only` and `.sr-only-focusable` to `.visually-hidden` and `.visually-hidden-focusable`.
- Renamed `sr-only()` and `sr-only-focusable()` mixins to `visually-hidden()` and `visually-hidden-focusable()`.

Reply:
I forgot that! Thanks Huong :+1: 
Also rebased and resolved conflicts